src/core/client/admin/components/PaginatedSelect/PaginatedSelect.tsx

```diff
@@ -66,42 +94,61 @@ const PaginatedSelect: FunctionComponent<Props> = ({
               )}
             </Dropdown>
           </IntersectionProvider>
-        </ClickOutside>
-      )}
-    >
-      {({ toggleVisibility, ref, visible }) => (
-        <Button
-          className={cn(styles.button, className)}
-          variant="flat"
-          adornmentLeft
-          color="mono"
-          onClick={toggleVisibility}
-          ref={ref}
-          uppercase={false}
-        >
-          {Icon && (
-            <ButtonSvgIcon className={styles.buttonIconLeft} Icon={Icon} />
-          )}
-          <Flex alignItems="center" className={styles.wrapper}>
-            {selected}
+        )}
+      >
+        {({ toggleVisibility, ref }) => (
+          <Flex
+            className={cn(styles.button, className)}
+            onClick={handleButtonClick}
+            ref={ref}
+            justifyContent="space-between"
+          >
+            {Icon && (
+              <ButtonSvgIcon className={styles.buttonIconLeft} Icon={Icon} />
+            )}
+            {open && !!onFilter ? (
+              <Localized id="admin-paginatedSelect-filter">
+                <TextField
```

Comment:
The styles for the text input seem a bit different from the design; for example, it has a border when focused and the text looks darker. Should these styles be updated?

Reply:
✅  I've updated the color of the text, however I think removing the outline on focus is an accessibility violation: https://www.a11yproject.com/posts/never-remove-css-outlines/#:%7E:text=Using%20the%20CSS%20rule%20%3Afocus,with%20the%20link%20or%20control.